templates/community-image-streams.json

```diff
@@ -115,6 +115,59 @@
                 ]
             }
         },
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "ubi8-openjdk-17",
+                "annotations": {
+                    "openshift.io/display-name": "Red Hat OpenJDK 17 (UBI8)",
+                    "openshift.io/provider-display-name": "Red Hat, Inc."
+                }
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "1.3",
```

Comment:
This segment describes ubi8/openjdk-17:1.3 but that version does not exist; the first version we have published is :1.10.

Reply:
(and in the other files)